src/routes/index.js

```diff
@@ -1,5 +1,9 @@
 const router = require('express').Router();
+const swagger = require('swagger-ui-express');
+const swaggerDoc = require('../../swagger.json');
 
 router.use('/api', require('./api'));
 
+router.use('/api/docs', swagger.serve, swagger.setup(swaggerDoc));
```

Comment:
on my knowledge, I think you should put line 7 before line 5. otherwise, it will never be reached since it has the same base path as line 5. so I suggest 
`router.use('/api/docs', swagger.serve, swagger.setup(swaggerDoc));
router.use('/api', require('./api'));
`

Reply:
It will. I think that would only happen if he had  `router.use('/api/:id', require('./api'));`

Reply:
so should I resolve or...

Reply:
I don't think it's important